src/pages/places/[placeId]/availability.test.js

```diff
@@ -121,19 +121,19 @@ test('The reason and error are cleared when switching back to "available"', asyn
     }),
   ).toBeDisabled();
 
-  userEvent.click(
+  await userEvent.click(
     screen.getByLabelText(nl.offerStatus.status.places.available),
   );
 
   expect(screen.queryByRole('alert')).not.toBeInTheDocument();
 
   expect(screen.getByLabelText(nl.offerStatus.reason).value).toBe('');
-});
+}, 10000);
```

Comment:
Try to avoid delay's.
Maybe just try adding `await` before the `expect` statement?